check_if_any.R

```diff
@@ -29,4 +29,18 @@ check_if_any  <- function( participant_codes, safe_query,ncit_code) {
  #   print("setting val to NO")
   }
   return(r)
-}
\ No newline at end of file
+}
+
+
+# Returns true ('YES') if check_if_any would return true for any parent of ncit_code.
+# In other words, starts looking one level up.
+check_if_any_parent <- function(participant_codes, safe_query, ncit_code) {
+  sql <- paste0("select parent from ncit_tc where descendant = $1")
```

Comment:
I think we need to just look one level up for parents - I think Mike said "look down and one level up".  So the one level down part would be "select parent from ncit_tc_with_path where descendant = $1 and level = 1" -- that gets us just the parents.  (there may be lots of levels above a node that we don't want to roll up to.).